packages/network/wireguard-linux-compat/patches/wireguard-linux-compat-01-skip-compat_h-version-checks.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/src/compat/compat.h b/src/compat/compat.h
+index 1aa0e7b..751ace7 100644
+--- a/src/compat/compat.h
++++ b/src/compat/compat.h
+@@ -43,7 +43,7 @@
+ #error "WireGuard requires Linux >= 3.10"
+ #endif
+ 
+-#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0)
++#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 7, 0)
```

Comment:
The version used in the line above conflicts with the version mentioned in the comment below - which is correct - is it `>= 5, 7, 0` or `>= 5.6`?

Currently `wireguard-linux-compat` doesn't build with 5.6-rc2 (it did build with 5.6-rc1) so I'm assuming the line above should `#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 6, 0)` and the comment is correct (in which case this patch shouldn't have been modified).

Reply:
If we don't want to build this package with kernel >= 5.6 then we shouldn't be building it... causing the build to fail is not the right fix.

Reply:
Temporary fix when building 5.6 for Geneirc:

https://github.com/LibreELEC/LibreELEC.tv/pull/4199/commits/436837112c24eafa9d83614a7374c650d26f0d2c